src/stories/index.tsx

```diff
@@ -1,7 +1,33 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
+import {
+  SmallWrapper,
+  MediumWrapper,
+  LargeWrapper,
+  XlargeWrapper
+} from "../components/Screens";
 import Example from "../components/Example";
+import Homepage from "../components/Home";
 import "../style/index.css";
 import "../style/fonts.css";
-
 storiesOf("Example", module).add("example", () => <Example />);
+storiesOf("Homepage", module).add("desktop version", () => (
```

Comment:
now we have addon in storybook for responsivity, so remove responsive versions

Reply:
fixed